### PR TITLE
Add Model to Django

### LINKED ISSUE #2: models
# Problem

We need to capture some conference attendees data.

# Proposed solution

Add the following models:

```
class Conference:
    conference_name = charfield
    conference_code = charfield

class Visit:
    name = charfield
    home = point
    date_left = datetime
    gravatar  = url 

```

### PR DESCRIPTION
**Pull request by euclidian**
Fix #2

### CODE REVIEW

**Comment by euclidian** [comment]
Fix #2

File: PyCon2017Namibia/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '=i(x(0rk(y7a0r%^w5*51ypr%cfyn#38@!#i8114(3k1!wix&d'
```
Comment (dimasciput) [inline PyCon2017Namibia/settings.py]: Don't put secret key in here.

Comment (meomancer) [reply PyCon2017Namibia/settings.py]: @euclidian move secret_key in other file (like secret.py) and add ignore for secret.py
see here : https://github.com/kartoza/docker-mezzanine/tree/develop/django_project/core/settings

**Comment by euclidian** [comment]
updated most project file please check @meomancer @dimasciput

File: deployment/docker/REQUIREMENTS.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+psycopg2
+Django>=1.8,<1.9
```
Comment (dimasciput) [inline deployment/docker/REQUIREMENTS.txt]: Change this to Django==1.10

File: django_project/core/settings/base.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+]
+
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.sites',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'suit',
```
Comment (dimasciput) [inline django_project/core/settings/base.py]: Remove this

File: deployment/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  restart: on-failure:5
+
+smtp:
+  # Note you cannot scale if you use conteiner_name
+  container_name: pycon-smtp
+  image: catatnight/postfix
+  hostname: postfix
+  environment:
+    # You could change this to something more suitable
+    - maildomain=terranex.co.za
```
Comment (meomancer) [inline deployment/docker-compose.yml]: change to kartoza.com

File: deployment/docker/REQUIREMENTS.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+django-suit==0.2.22
```
Comment (meomancer) [inline deployment/docker/REQUIREMENTS.txt]: remove this

File: django_project/core/settings/project.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+# Set debug to false for production
+DEBUG = TEMPLATE_DEBUG = False
+
+SOUTH_TESTS_MIGRATE = False
+
+# Set languages which want to be translated
+LANGUAGES = (
+    ('en', _('English')),
+    ('en', _('Latin')),
```
Comment (meomancer) [inline django_project/core/settings/project.py]: remove this unless we need other language

**Pull request merged**